packages/table/stories/table-virtualized.stories.ts

```diff
@@ -200,6 +202,7 @@ virtualizedSingle.args = {
 export const virtualizedMultiple = (args: Properties): TemplateResult => {
     return html`
         <sp-table
+            emphasized
             size="m"
```

Comment:
if not size is mentioned what would be the default?

Reply:
In CSS, all medium styles display as default.